packages/orbit-design-tokens/src/index.ts

```diff
@@ -1,3 +1,5 @@
+import { parseToRgba } from "color2k";
```

Comment:
I don't think we should be doing this. We are reexporting this inside our package just to be used in another package. If that's the case, the package that needs it should import it.

Reply:
this isn't working like that, in case if the package will import this -> it will need to have it as dependency. I do not want to add additional dependency to tailwind preset. It suits tokens API, where we export those functions like for rgba and transparent color

Reply:
> I do not want to add additional dependency to tailwind preset.

It is actually being used in `orbit-components`. And I don't agree with this "masking" dependency. If it's needed there, it's a dependency there. It doesn't feel right to be reexporting functions from other packages.

Either that or we find a way to export the data already processed (I'm not sure if it's possible) so we don't use the function there.

Reply:
👍🏻  can you give more arguments here why it's not right in your opinion? I feel like we do not have strong arguments against it. Reexporting is fine in this case imo, we have a tokens package and related functions to it, if the function is used in the tokens package and reexported -> it's fine to me, I do not see much value in adding it on both places (tailwind and components) and adding additional dep to components because of single usage that might be removed later :) 

We are reexporting a single function, that will be used on two other packages. I am not sure about how much it will increase bundle size, if we will add this package as a dependency to other two packages, I am also not comfortable with adding a new dependency because of single function, that might be removed later :) 

This solution looks less invasional to me, wdyt? 🙂